tsfc/geometric.py

```diff
@@ -155,6 +163,11 @@ def translate_reference_cell_volume(terminal, mt, params):
     return gem.Literal(reference_cell_volume[reference_cell(terminal)])
 
 
+@translate.register(ReferenceFacetVolume)
+def translate_reference_facet_volume(terminal, mt, params):
```

Comment:
This (and the previous two snippets) is only used if there is no `FacetArea` in `preserve_geometry_types=` when calling `compute_form_data`.